### DISCUSSION TITLE
Strelka question

### DISCUSSION THREAD

**Discussion by ktroberson** [Security-Onion-Solutions/securityonion#7979] [Unsupported Versions]
working an issue in my Sec Onion where an executable was delivered via email, it was extracted by zeek and was label SMTP<randomname>.exe. But it does seem like it was processed by strelka. I have looked in process and unprocess directories. Am I missing something? I thought strelka ingested all excutables that are extracted.

**Comment by ktroberson** [comment]
actually none of the executables are being processed by strelka

**Comment by dougburks** [comment]
Is Strelka enabled and running?

Have you checked the Strelka logs for additional clues?
https://docs.securityonion.net/en/2.3/strelka.html#diagnostic-logging
